src/modules/ipfix/aggregator/BaseHashtable.cpp

```diff
@@ -480,6 +480,12 @@ void BaseHashtable::postReconfiguration()
 	sendDataTemplate();
 }
 
+void BaseHashtable::clearStatistics()
+{
+	statExportedBuckets = 0;
+	statLastExpBuckets = 0;
+}
```

Comment:
Some stat are related to the current state of the table (ie: how many items it contains), so I think it would be wrong to reset those. Were you thinking about some specific one?